### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by taiwo2**
In this milestone, I have created components for the Calculator app. They are App, Button, ButtonPanel and Display components which are done with their styliing

### CODE REVIEW

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      <div className="top-btn">
+        <Button name="1" />
+        <Button name="2" />
+        <Button name="3" />
+        <Button name="+" />
+      </div>
+      <div className="top-btn">
+        <Button name="0" />
+        <Button name="." />
+        <Button name="=" />
```
Comment (bellom) [inline src/components/ButtonPanel.js]: ### [Optional]
Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code, In other words; you repeated the `Button` component multiple times.

You can use [objects.key](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)  and `map` to loop through the objects. `objects.keys` for the outer loop and `map` for the inner loop.

Example; 
```
const GroupButtons = {
  Group 1: ['AC', '+/-', '%', '÷'],
  Group 2: ['7', '8', '9', 'X'],
   ............
}
```
OR

[Creating a two-dimensional array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#creating_a_two-dimensional_array) and implement nested loop with a map.
EXample;
```
  const buttons = [
    ['AC', '+/-', '%', '÷'],
    ['7', '8', '9', 'X'],
    ..........
  ];
```

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    </div>
+  );
+};
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
+  result: 0,
```
Comment (bellom) [inline src/components/Display.js]: This defaultProps should be a string, as you specified on line 14.
That is; `result: '0'`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @\ :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by afanimartin — APPROVED** [review]
## Approved ✅ ✅ ✅ 
Nice work making the requested change. 
Merge and proceed to the next milestone.

### Martin Afani
#### Slack: whiz25

**Pull request merged**